tcms/templates/case/edit.html

```diff
@@ -1,10 +1,11 @@
 {% extends "tcms_base.html" %}
+{% load i18n %}
 {% load static %}
 {% block subtitle %}
     {% if test_case %}
-        Edit test case - {{ test_case.summary }}
+        {% trans "Edit test case" %} - {{ test_case.summary }}
     {% else %}
-        Add new test case
+        {% trans "Add new test case" %}
```

Comment:
FYI this file will be removed as part of #730 which will also be marked for translations so you probably want to exclude these changes and save yourself from resolving a conflict later.

Reply:
ok

Reply:
Hi @atodorov 
I've rebase from master, #730 is merge but [case/edit.html](https://github.com/kiwitcms/Kiwi/blob/master/tcms/templates/case/edit.html) have not been removed